src/hooks/useConnectorImages.ts

```diff
@@ -1,44 +1,26 @@
-import { useEffect, useState } from 'react';
-import axios, { withAxios } from 'services/axios';
-import { BaseHook } from 'types';
+import { ConnectorImageData, connectorsEndpoint } from 'endpoints/connectors';
+import { useAsync } from 'hooks/useAsync';
+import { useEffect } from 'react';
 
-interface ConnectorImagesService extends BaseHook {
-    data: {
-        attributes: any;
-        links: any;
-    };
-}
-
-const useConnectorImages = (
-    imagesURL: string,
-    whichOne: number = 0
-): ConnectorImagesService => {
-    const [loading, setLoading] = useState<boolean>(true);
-    const [error, setError] = useState<string | null>(null);
-
-    const [attributes, setAttributes] = useState<object>({});
-    const [links, setLinks] = useState<object>({});
+const useConnectorImages = (imagesURL: string, whichOne: number = 0) => {
```

Comment:
The name `whichOne` does not sit well with me. If there is room to adjust it, would something along the lines of `selection` or `selectedIndex` work?

Reply:
Yup... I don't really have strong feelings on this naming.